modules/custom/az_finder/css/taxonomy-index-tid-widget.css

```
@@ -33,66 +40,98 @@
 .js-svg-replace-level-1 svg {
   pointer-events: none;
   display: flex;
-  min-width: 16px;
 }
 
 .js-svg-replace-level-0 svg {
   margin-left: auto;
   min-width: 24px;
 }
 
+.js-svg-replace-level-1 svg {
+  margin-left: -1.5px;
+  margin-right: 5.5px;
+  min-width: 16px;
+}
+
 .az-bef-vertical fieldset {
   min-width: 100%;
 }
 
-.az-bef-vertical .collapser {
+/*
+  Note: Using bottom margins and padding within .az-bef-checkboxes may affect
+  the smoothness of the collapse and expand animations.
+*/
+.az-bef-checkboxes .checkbox {
+  width: 100% !important;
+  display: flex;
+  align-items: center;
+  line-height: 1.25rem;
+  margin-top: 0.75rem;
+  margin-bottom: 0.125rem;
+}
+
+.az-bef-checkboxes .collapser {
   font-weight: 500;
 }
 
-.az-bef-vertical .collapser:not(.level-0) {
+.az-bef-checkboxes .collapser:not(.level-0) {
   color: unset !important;
 }
 
-.az-bef-vertical li:has(.collapser.level-0) {
-  border-bottom: 1px solid #e9ecef;
+.az-bef-checkboxes li:has(.collapser.level-0) {
   padding: 1rem 0;
+  border-top: 1px solid #e9ecef;
+}
+
+.az-bef-checkboxes > li:not(:has(.collapser)) + li:has(.collapser) {
+  margin-top: 1.125rem;
+}
+
+.az-bef-checkboxes > li:first-child:not(:has(.collapser)),
+.az-bef-checkboxes > li:has(.collapser) + li:not(:has(.collapser)) {
+  padding-top: 0.25rem;
+  border-top: 1px solid #e9ecef;
 }
 
-.az-bef-ul>li:has(.collapser) {
-  margin-top: 0.5rem;
+.az-bef-checkboxes > li:last-child {
+  margin-bottom: 1rem;
 }
 
-.az-bef-ul>li:has(.collapser) li {
-  margin-top: 0.25rem;
+.az-bef-checkboxes > li:last-child:has(.collapser) {
+  margin-bottom: 0.25rem;
 }
 
 .az-bef-checkboxes>li>.checkbox {
-  display: block;
-  margin-top: .5rem;
   font-size: 1rem;
   font-weight: 500;
   color: #1e5288;
 }
 
 .az-bef-checkboxes .form-checkbox {
-  margin-top: 0.25rem;
+  display: flex;
+  vertical-align: middle;
+  margin-top: 0;
+  margin-bottom: 0;
 }
 
 .az-bef-checkboxes>li>ul>li>.checkbox {
   font-weight: 500;
 }
 
 .az-bef-checkboxes ul>li>ul {
-  padding-left: 20px;
+  padding-left: 24px;
 }
 
 .az-bef-checkboxes>li>ul>li>ul {
-  padding-left: 15px;
+  padding-left: 18px;
```

```suggestion
  padding-left: 22px;
```

Looks like this being inconsistent with the `.az-bef-checkboxes ul>li>ul` is visually noticeable. I recommend making them the same (or combined for condensed-ness).

---

Thanks! Yep, if we want that consistent padding value for the indentation then we don't need this extra CSS rule.